cmd/saml2aws/main.go

```diff
@@ -86,6 +87,7 @@ func main() {
 	cmdLogin.Flag("client-id", "OneLogin client id, used to generate API access token. (env: ONELOGIN_CLIENT_ID)").Envar("ONELOGIN_CLIENT_ID").StringVar(&commonFlags.ClientID)
 	cmdLogin.Flag("client-secret", "OneLogin client secret, used to generate API access token. (env: ONELOGIN_CLIENT_SECRET)").Envar("ONELOGIN_CLIENT_SECRET").StringVar(&commonFlags.ClientSecret)
 	cmdLogin.Flag("force", "Refresh credentials even if not expired.").BoolVar(&loginFlags.Force)
+	cmdLogin.Flag("quiet", "Supress credentials not expired message").BoolVar(&loginFlags.Quiet)
```

Comment:
May this should just be "Suppresses information messages" as it'd be neat if this could silence the `login` output if you `--skip-prompt`.

Reply:
Fully Agree on the message.
On the --skip-prompt. when login is required, I'm not sure is a good idea, at least I would leave the 'Waiting for approval, please check your XXXX Verify app' message visible.

Reply:
Changes done, please my apologies if the way is not the best, I'm not a programmer just a DevOps/SRE/SysAdmin :)
hopefully it is good enough.